test/decl/overload.swift

```diff
@@ -149,8 +149,15 @@ func ovl_generic3<T : P2>(_ x: T) {} // OK
 struct X { }
 struct Y { }
 struct Z { 
-  var a : X, // expected-note{{previously declared here}}
-  a : Y // expected-error{{invalid redeclaration of 'a'}}
```

Comment:
Please don't remove this example; it may be relevant that two things are in the same pattern-binding. You can make a new struct instead if you want to test the other case.